pkg/blackduck/certificate.go

```diff
@@ -116,3 +120,78 @@ func CreateSelfSignedCert() (string, string) {
 	pem.Encode(key, pemBlockForKey(priv))
 	return certificate.String(), key.String()
 }
+
+func GetCertificateSecretFromFile(secretName, namespace, certPath, keyPath string) (*corev1.Secret, error) {
```

Comment:
can we move this into the helm chart itself? ... maybe not in this PR, but if we're ok with that, I'll make a ticket to track it